bindings/gumjs/gumscript-runtime-objc.js

```diff
@@ -389,6 +413,15 @@
                         case "valueOf":
                             const description = target.description();
                             return description.UTF8String.bind(description);
+                        case "equals":
+                            return function equals (ptr) {
```

Comment:
How about moving this function down next to the others and just returning it here?